openupgrade_scripts/apriori.py

```diff
@@ -79,6 +79,8 @@
     "coupon_commercial_partner_applicability": "loyalty_partner_applicability",
     # OCA/sales-workflow
     "sale_product_set_layout": "sale_product_set",
+    # OCA/pos
```

Comment:
Please insert this in alphabetically order

Reply:
The original file is not ordered correctly. I fixed it.

Reply:
@legalsylvain Nice PR. Thanks for the work!